graal/backends/core/analyzers/cloc.py

```diff
@@ -83,14 +80,12 @@ def __analyze_repository(self, message):
                 if line.lower().startswith("sum"):
                     break
                 elif not line.startswith("-----"):
-                    digested = " ".join(line.split())
```

Comment:
Also, I found this logic redundant which existed for the longest time and fixed it. Let me know what you think!